book/website/reproducible-research/reviewing/reviewing-motivation.md

```diff
@@ -43,13 +49,28 @@ This kind of feedback on documentation applies equally to user-facing documentat
 Related to documentation, code review can also help to ensure that code is readable and easy to understand. Having a second pair of eyes can help spot areas where the code might be difficult to follow.
 The more readable your code is, the easier it will be for other developers to reproduce your code for their own purposes.
 
+## Deployability
+
+One of the most challenging tasks in software engineering is ensuring code can be successfully built and run across a range of different environments.
+The operating system, package manager, window manager, existing dependencies, installed software, localisation and individual settings all contribute to the environment within which a piece of software runs.
+There have been many attempts, from packaging to containerisation, to reproducible builds {cite:ps}`Builds2023reprobuilds` and reproducible build environments {cite:ps}`Choi2023reproenvs`, to constrain the environment in order to make deployment easier and more consistent.
+
+It remains an unsolved problem.
+For open source projects especially, having changes built and run in a mixture of different environments by different developers is therefore a valuable exercise.
+Reviewers should aim to always build and run changes themselves.
+This is a test for the robustness of the changes to different environments, robustness of the build system and also the documentation for setting things up.
+
 (rr-reviewing-motivation-enforcement)=
 ## Style Enforcement
 
 Many projects enforce certain {ref}`code style guidelines<rr-code-quality>`, be they widely-adopted standards (for example, [PEP8](https://www.python.org/dev/peps/pep-0008/), the [Google C++ style guide](https://google.github.io/styleguide/cppguide.html)) or more project-specific conventions. 
 {ref}`Automated services<pd-code-styling-tools>` provide a convenient way to enforce a coding style and start the discussion about code quality.
 
-Code review provides an opportunity to ensure all proposed changes meet the minimum required standards for the project.
+While it's important to automate as many checks as possible, it's worth noting that for many projects the style requirements extend beyond those enforced through automated checks.
```

Comment:
```suggestion
While it is important to automate as many checks as possible, it is worth noting that for many projects the style requirements extend beyond those enforced through automated checks.
```
Ironically, the changes suggested here are also stylistic rather than substantive 😄

Reply:
:laughing: